test/DurableTaskEndToEndTests.cs

```diff
@@ -585,6 +585,60 @@ public async Task TerminateOrchestration(bool extendedSessions)
             }
         }
 
+        /// <summary>
+        /// End-to-end test that validates the Rewind functionality.
+        /// </summary>
+        [Theory]
+        //[InlineData(true)]
+        [InlineData(false)]

```

Comment:
I suggest we remove the `[InlineData(...)]` attributes and just hardcode `extendedSessions` to `true` inside the method.